replaced service chain readme (#391)
klaytn/klaytn-docs: docs/node/service-chain/references/README.md

> # Service Chain <a id="service-chain"></a>

Reviewer: ```suggestion
# ServiceChain <a id="service-chain"></a>
```

## Intended Audience <a id="intended-audience"></a>

- Companies that want to build blockchains for Metaverse, GameFi, and NFTs
- dApp developers who need high TPS, minimal transaction fees, and data privacy.  
- Anyone who wants to build a local private network or a ledger database for testing.

> ## Service Chain Overview <a id="service-chain-overview"></a>

Reviewer: ```suggestion
## ServiceChain Overview <a id="service-chain-overview"></a>
```

> Klaytn Service Chain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn Service Chain provides the following features:

Reviewer: ```suggestion
Klaytn ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
```

- Immediate finality
- Cross-chain token transfer
- Data anchoring to the main chain for data integrity
- Multi-sig bridge contract to meet enterprise-level security requirements

> ![](../../klaytn/images/sc_serviceChains.png)

Reviewer: ```suggestion
![](./images/sc_serviceChains.png)
```
I request changing the image path.

> Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain.

Reviewer: ```suggestion
Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain. And the following videos will help you understand Klaytn ServiceChain.

[Horizontal Scaling through Service Chain in Klaytn | TXGX 2019](https://www.youtube.com/watch?v=8yQc5FQysJc)
[High Availability Architecture of Klaytn Service Chain | TXGX 2019](https://www.youtube.com/watch?v=HcdhWtXPuR0)
```

> Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the Service Chain.

Reviewer: ```suggestion
Read the [Klaytn Scaling Solution](../../klaytn/scaling-solutions.md) for more details about the ServiceChain.
```

# ServiceChain <a id="service-chain"></a>

## Intended Audience <a id="intended-audience"></a>

- Companies that want to build blockchains for Metaverse, GameFi, and NFTs
- dApp developers who need high TPS, minimal transaction fees, and data privacy.  
- Anyone who wants to build a local private network or a ledger database for testing.

## ServiceChain Overview <a id="service-chain-overview"></a>

> Klaytn ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:

Reviewer: ```suggestion
ServiceChain is an enterprise-level blockchain to meet companies' requirements such as value transfer, security, high performance, and customization. Klaytn ServiceChain provides the following features:
```

- Immediate finality

> - Cross-chain token transfer

Reviewer 2: Hmm can it be misunderstood to readers that ServiceChain feature can transfer tokens between any other blockchains?

Token transfer between ServiceChains?
Token transfer between Klaytn chains?

Reviewer: I suggest the revised phrase in the review below.

Reviewer: @kjhman21 
I am sorry 
I understood it as a way to collaborate like document editing in Google Docs,
In the future, we will leave it without resolving.

> - Cross-chain token transfer

Author: ```suggestion
- Token transfer between Klaytn chains
```
